README.md

```diff
@@ -1,2 +1,13 @@
 # soton-analytica-display
 Gotta go FAST.
+
+
+## Requirements
+
+- Python 3
+- `pipenv` - `pip3 install pipenv`
+
+## Recommended packages
+
+- LibYAML
+    - `libyaml-dev` on Ubuntu APT
```

Comment:
Documentation is a security risk. I propose security through obscurity as a solution.